source/installation-notes.rst

```diff
@@ -60,6 +60,29 @@ On a Debian (and presumably Ubuntu) system, enter the following:
    rustup target add x86_64-unknown-linux-musl
    cargo build --target=x86_64-unknown-linux-musl --release
 
+Using tmpfs for the RPKI Cache
+------------------------------
+
+The full RPKI data set consists of hundreds of thousands of small files. This
+causes a considerable amount of disk I/O with each validation run. In some cases
```

Comment:
Are we able to say which cases rather than "some" cases? Otherwise readers have to work harder to decide if this applies to them or not.

Reply:
I can't decide for an operator if a lor of disk I/O is undesirable in their specific setup, so I kept it generic.